<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" >
 <author>rncbc aka Rui Nuno Capela</author>
 <comment>qtractor - An Audio/MIDI multi-track sequencer.

   Copyright (C) 2005-2024, rncbc aka Rui Nuno Capela. All rights reserved.

   This program is free software; you can redistribute it and/or
   modify it under the terms of the GNU General Public License
   as published by the Free Software Foundation; either version 2
   of the License, or (at your option) any later version.

   This program is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU General Public License for more details.

   You should have received a copy of the GNU General Public License along
   with this program; if not, write to the Free Software Foundation, Inc.,
   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.

 </comment>
 <class>qtractorPluginSelectForm</class>
 <widget class="QDialog" name="qtractorPluginSelectForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Plugins</string>
  </property>
  <property name="windowIcon">
   <iconset theme="pluginEdit" />
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>4</number>
   </property>
   <property name="spacing" >
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>4</number>
     </property>
     <item>
      <widget class="QToolButton" name="PluginResetToolButton" >
       <property name="minimumSize" >
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="toolTip" >
        <string>Reset filter</string>
       </property>
       <property name="text" >
        <string>X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="PluginSearchComboBox" >
       <property name="minimumSize" >
        <size>
         <width>320</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>Plugin search string (regular expression)</string>
       </property>
       <property name="editable" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="PluginTypeComboBox" >
       <property name="maximumSize" >
        <size>
         <width>80</width>
         <height>22</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>Plugin type</string>
       </property>
       <property name="editable" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="PluginListView" >
     <property name="minimumSize" >
      <size>
       <width>420</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip" >
      <string>Available plugins</string>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="rootIsDecorated" >
      <bool>false</bool>
     </property>
     <property name="uniformRowHeights" >
      <bool>true</bool>
     </property>
     <property name="itemsExpandable" >
      <bool>false</bool>
     </property>
     <property name="allColumnsShowFocus" >
      <bool>true</bool>
     </property>
     <property name="sortingEnabled" >
      <bool>true</bool>
     </property>
     <column>
      <property name="text" >
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Audio</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>MIDI</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Control</string>
      </property>
     </column>
     <column>
      <property name="text" >
	   <string>Modes</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Path</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Index</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Instances</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Type</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>4</number>
     </property>
     <item>
      <widget class="QPushButton" name="PluginRescanPushButton" >
       <property name="toolTip" >
        <string>Rescan for available plugins (refresh)</string>
       </property>
       <property name="text" >
        <string>&amp;Rescan</string>
       </property>
       <property name="icon" >
        <iconset theme="formRefresh" />
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QProgressBar" name="PluginScanProgressBar" >
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>240</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>Plugin scanning in progress...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="DialogButtonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="4" />
 <tabstops>
  <tabstop>PluginResetToolButton</tabstop>
  <tabstop>PluginSearchComboBox</tabstop>
  <tabstop>PluginTypeComboBox</tabstop>
  <tabstop>PluginListView</tabstop>
  <tabstop>PluginRescanPushButton</tabstop>
  <tabstop>DialogButtonBox</tabstop>
 </tabstops>
 <resources>
  <include location="qtractor.qrc" />
 </resources>
 <connections/>
</ui>
